benchmarks/shared/src/main/scala/higherkindness/mu/rpc/benchmarks/shared/protocols/PersonServicePB.scala

```diff
@@ -21,6 +21,10 @@ package protocols
 import higherkindness.mu.rpc.benchmarks.shared.models._
 import higherkindness.mu.rpc.protocol._
 
+import cats.implicits._
+import cats.derived._
+import auto.monoid._
+
```

Comment:
Do the users need to include these imports in all the protocols annotated with `@service(Protobuf)`?
I'm wondering why don't we just inject them in the macro, only in the case that `serializationType == Protobuf`.

Reply:
That sounds like a good idea

Reply:
So I [attempted this today](https://github.com/higherkindness/mu/compare/606-default-typeclass...dzanot:606-default-typeclass?expand=1#diff-90fc855999aa8ba5dc1873744eb5d339L29) and ran into some...very weird behavior. Moving the imports to the `service()` macro *mostly* worked, except in the case of `PersonServicePB`. I narrowed down the issue to the fact that its `PersonList` message contains a `List`. 

That is: `PersonServicePB` compiled with the kittens import in the `.scala` file but _not_ when added by the macro impl. All the other cases of `service(Protobuf)` were able to compile in both cases. 

So I wrote some sanity tests to prove that kittens can derive instances for such case classes, and discovered some other very strange behavior: To get the auto derivation to work and the test to compile I had to `include cats.implicits._` a _second_ time [as you can see here](https://github.com/higherkindness/mu/compare/606-default-typeclass...dzanot:606-default-typeclass?expand=1#diff-f749bcf748f83ddc00f394b3558820e9R90).

This discovery didn't solve the initial problem but it revealed to me some quirkiness that is beyond my understanding. Does anyone have any experience with multiple imports being necessary?

Reply:
I've never come across multiple imports being necessary, is this to do with it being in scope for the generated code but then not here unless it's imported?

Reply:
The test where multiple imports is "necessary" is independent of the generated code. It's testing the Monoid->ProtoDefault->Marshaller chain. As far as I can tell when I am importing the kittens implicits in the macro/generated code kittens isn't actually able to derive any monoid instances.

Reply:
FWIW I resolved the macro issues, duplicate instances of...instances were being imported because `pbd` inherited some https://github.com/dzanot/mu/commit/9ae27508501a5bf4993892be4e0e6de18079892f

Reply:
Congrats on resolving this @dzanot! 🎉

Reply:
Oh nice! Thanks @dzanot!